core/application/Models/dbModels/NotificationModel.ts

```diff
@@ -21,37 +21,43 @@ export default class NotificationModel implements NosqlModel<NotificationFields>
 
   fromDto(notificationDto: NotificationDTO): NotificationFields {
     const { id, userId, type, data, ...remainingNotificationFields } = notificationDto
-    let sk = `COMMON#${id}`
     if (type === 'bloodRequestPost' && data !== undefined && data.requestPostId !== undefined) {
-      sk = `BLOODREQPOST#${data.requestPostId}`
-    }
-
-    return {
-      PK: `NOTIFICATION#${userId}`,
-      SK: sk,
-      ...remainingNotificationFields,
-      createdAt: new Date().toISOString()
+      return {
+        PK: `NOTIFICATION#${userId}`,
+        SK: `BLOODREQPOST#${data.requestPostId}`,
+        ...remainingNotificationFields,
+        data,
+        createdAt: new Date().toISOString()
+      }
+    } else {
```

Comment:
no need for else

Reply:
Updated in 8e0ceed51f83432c99f1e74537ac5a9c0d070e61